### PR TITLE
Mandos nft metadata

### PR DESCRIPTION
**Pull request by dorin-iancu**
Added the rest of metadata fields for NFTs:
- creator
- royalties
- hash
- URI
- attributes

"Name" was not added, as it's not relevant for testing and it can be confused with the identifier.

### CODE REVIEW

File: arwenmandos/stepCheckAccounts.go [hunk 1]
```diff
@@ -274,8 +274,49 @@ func checkTokenInstances(
 					expectedInstance.Balance.Original,
 					accountInstance.Value))
 			}
+			if !bytes.Equal(expectedInstance.Creator.Value, accountInstance.TokenMetaData.Creator) {
```
Comment (andrei-marinica) [inline arwenmandos/stepCheckAccounts.go:277]: Use:
`expectedInstance.Creator.Check(accountInstance.TokenMetaData.Creator)`
To also allow for the check to be ignored, when the developer writes `"*"` .
The same goes for all the other checks below.

**Review by andrei-marinica — CHANGES_REQUESTED** [review]

File: arwenmandos/stepCheckAccounts.go (hunk 1 above)
Comment (dorin-iancu) [reply arwenmandos/stepCheckAccounts.go]: Done

**Review by andrei-marinica — APPROVED** [review]

**Review by andrei-marinica — DISMISSED** [review]

**Review by andrei-marinica — APPROVED** [review]

File: mock/world/mockESDTAccount.go [hunk 2]
```diff
@@ -249,18 +250,25 @@ func (a *Account) loadMockESDTDataInstance(tokenKey []byte) (string, *esdt.ESDig
...
-		tokenNameFromKey := GetTokenNameFromKey(tokenKey)
 		tokenInstance.TokenMetaData = &esdt.MetaData{
 			Name:  tokenNameFromKey,
 			Nonce: 0,
 		}
 		tokenName = string(tokenNameFromKey)
 	} else {
-		// the key also contains the nonce, we take the token identifier from the metadata
-		tokenName = string(tokenInstance.TokenMetaData.Name)
+		nonceAsBytes := big.NewInt(int64(tokenInstance.TokenMetaData.Nonce)).Bytes()
```
Comment (andrei-marinica) [inline mock/world/mockESDTAccount.go:264]:
```suggestion
		nonceAsBytes := big.NewInt(0).SetUint64(tokenInstance.TokenMetaData.Nonce).Bytes()
```

Comment (dorin-iancu) [reply mock/world/mockESDTAccount.go]: Fixed.

**Review by andrei-marinica — APPROVED** [review]

**Review by catalinnnn — APPROVED** [review]

**Pull request merged**